components/molecules/card.tsx

```diff
@@ -0,0 +1,15 @@
+import Icon from '../atoms/icon'
+
+export default function Card ({ icon, title, list }) {
```

Comment:
You need to create type to your components props otherwise you will have lot of linter issues when you will have to merge with my branch